include/database/DBManager.php

```diff
@@ -4039,4 +4039,9 @@ public function enableKeys($tableName)
      * @return string
      */
     abstract public function getGuidSQL();
+
+
+    function removeLineBreaks($sql) {
```

Comment:
Please state the visibility of the function explicitly (private/public)

Reply:
@PedroErnst  Done!